plugins/flashrom/fu-flashrom-internal-device.c

```diff
@@ -156,7 +157,13 @@ fu_flashrom_internal_device_write_firmware(FuDevice *device,
 	}
 	flashrom_layout_release(layout);
 
+	/* Check if CMOS needs a reset */
+	if (fu_device_has_private_flag(device, FU_FLASHROM_DEVICE_FLAG_RESET_CMOS)) {
+		g_debug("Attempting CMOS Reset");
+		return fu_reset_cmos();
```

Comment:
you need to be passing error into this function.  Otherwise a FALSE return will not set an error.